PresentScreenings/Controllers/ViewController.cs

```diff
@@ -652,7 +652,7 @@ public void MoveScreening(bool forward)
             {
                 onDemandScreening.MoveStartTime(GetSpanToFit(onDemandScreening, forward));
                 Plan.InitializeDays();
-                UpdateAttendanceStatus(onDemandScreening);
+                UpdateDayAttendanceStatus(onDemandScreening);
```

Comment:
Please consider check a minimal number of screenings, in the line of the change in method `MoveScreening24Hours`.